packages/react-ui-core/src/Form/LabelHidingField.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import autobind from 'autobind-decorator'
+import Field from './Field'
+
+@themed(/^LabelHidingField/)
+export default class LabelHidingField extends Component {
+  static propTypes = {
+    theme: PropTypes.object,
+    name: PropTypes.string,
+    type: PropTypes.string,
+    label: PropTypes.string,
+    value: PropTypes.string,
+    defaultValue: PropTypes.string,
+    onChange: PropTypes.func,
+    nodeType: PropTypes.any,
```

Comment:
I don't think `any` will work here. An object would break this. You could do:
```
    nodeType: PropTypes.oneOfType([
      PropTypes.node,
      PropTypes.func,
      PropTypes.object,
    ]),
```
and apply object and such to Field or just make it just a:
```
  nodetype: PropTypes.node
```
??